guide/rotterdam/template_zhuoyu.qmd

```diff
@@ -0,0 +1,17 @@
+---
+title: "recommendation for visiting Rotterdam"
+description: "walk along the Nieuwe maas river"
```

Comment:
This should be the `title`. Maybe you can think of something else for the description?